src/modules/did/did.service.ts

```diff
@@ -309,6 +309,16 @@ export class DIDService implements OnModuleInit {
     }
   }
 
+  public async resolveServiceEndpoints(did: string) {
```

Comment:
Could you add documentation and return type to this method please? Does this seem valuable from your perspective?

Reply:
Sure, I'll try to find eslint rule for this